docs/resources/resources.md

```diff
@@ -0,0 +1,39 @@
+---
+id: resources
+title:  "Resources"
+---
+
+## Blog Articles
+
+Note that these articles reflect the state of ZIO at the time of their publication, so the code samples might be outdated considering ZIO is still under active development.
+- [Bifunctor IO: A Step Away from Dynamically-Typed Error Handling](http://degoes.net/articles/bifunctor-io) by John De Goes
+- [Beautiful, Simple, Testable Functional Effects for Scala](http://degoes.net/articles/zio-environment) (introducing ZIO Environment) by John De Goes
+- [Testing Incrementally with ZIO Environment](http://degoes.net/articles/testable-zio) by John De Goes
+- [ZIO & Cats Effect: A Match Made in Heaven](http://degoes.net/articles/zio-cats-effect) by John De Goes
+- [Elevator Control System using ZIO](https://medium.com/@wiemzin/elevator-control-system-using-zio-c718ae423c58) by Wiem Zine Elabadine
+
+## Talks
+
+- [ZIO: Next-Generation Effects in Scala](https://www.youtube.com/watch?v=mkSHhsJXjdc&t=6s) by John De Goes
+- [ZIO Schedule: Conquering Flakiness and Recurrence with Pure Functional Programming](https://www.youtube.com/watch?v=onQSHiafAY8&t=1s) by John De Goes
+- [Scalaz Stream: Rebirth](https://www.youtube.com/watch?v=mLJYODobz44&t=15s) by John De Goes and Itamar Ravid
+- [ZIO Queue: A new Queue for a new Era](https://www.youtube.com/watch?v=8JLprl34xEw&t=2437s) by John De Goes
+- [The Death of Tagless Final](https://skillsmatter.com/skillscasts/13247-scala-matters) by John De Goes
+- [Atomically { Delete Your Actors }](https://www.youtube.com/watch?v=d6WWmia0BPM) by John De Goes and Wiem Zine Elabadine
+- [Error Management: Future vs ZIO](https://www.youtube.com/watch?v=mGxcaQs3JWI) by John De Goes and Kai
+- [ZIO Queue](https://www.youtube.com/watch?v=lBYkLc-j7Vo) by Wiem Zine Elabadine
+- [Tour of ZIO](https://www.youtube.com/watch?v=5s0GOA3WQnY&t=1405s) by Oleksandra Holubitska
+
```

Comment:
The oldest talks were Scale by the Bay and Scala IO, on the SZ8 IO Monad, but they may be too dated.

Reply:
Yeah, I kept them out because they're quite old and a lot of things changed. Also all the recent ones re-introduced the main concepts.